fix: jwt-expiry

23000003 opened the pull request:
# Description

jwt expiry should validate if its between 0 and the queue start time. 

## Type of change

## Checklist before requesting a review

- [X] I have performed a self-review of my code

Luzefiru reviewed (changes requested):
Can we simplify this and have less reliance on multiple `.env` variables?

For example, a function that generates an expiry based on the end time like `getJwtExpiry("17:00") => number`, where `17:00` is `5pm` and `number` is how many milliseconds until `5pm` today.

23000003 commented:
> Contributor

apologies,.. i refactored it na, i added the queue start in case someone asks for a token before it starts,, this the only solu i can think pero ill accept any feedbacks!

Luzefiru reviewed (approved):
LGTM.
